### PR TITLE
add installing software and troubleshooting section for ISC'22 tutorial

### PR DESCRIPTION
**Pull request by SebastianAchilles**

### CODE REVIEW

File: docs/2022-isc22/troubleshooting.md [hunk 1]
```diff
@@ -1 +1,485 @@
...
+```shell
+export EASYBUILD_PREFIX=$HOME/easybuild
+export EASYBUILD_BUILDPATH=/tmp/$USER
+```
+
+Check your configuration via `eb --show-config`.
+
+Strictly speaking the configuration doesn't matter much for the sake of this
+exercise, but it may help with the step-wise approach we'll take and
+grasping the solutions.
```
Comment (klust) [inline docs/2022-isc22/troubleshooting.md:277]: I wonder if we should add a warning that depending on how EasyBuild is used, a user may actually bump into T.2 before T.1? If you have the automatism to first check for dependencies, e.g., with eb -D, you'll actually first run in T.2, but if you just do `eb subread.eb` you'll run in T.1 first. Just a thought though, not a necessity.

Comment (klust) [inline docs/2022-isc22/troubleshooting.md:277]: I wonder if we should add a warning that depending on how EasyBuild is used, a user may actually bump into T.2 before T.1? If you have the automatism to first check for dependencies, e.g., with eb -D, you'll actually first run in T.2, but if you just do `eb subread.eb` you'll run in T.1 first. Just a thought though, not a necessity.

File: docs/2022-isc22/troubleshooting.md [hunk 2]
```diff
@@ -1 +1,485 @@
...
+    The installation fails because the easyconfig specifies that GCC 8.5.0
+    should be used as toolchain:
+    ```
+    $ eb subread.eb
+    ...
+    == FAILED: Installation ended unsuccessfully (build directory: /tmp/easybuild/Subread/2.0.1/GCC-8.5.0): build failed (first 300 chars):
+    No module found for toolchain: GCC/8.5.0 (took 1 sec)
+    ```
+
+    We don't have this GCC version installed, but we do have GCC 10.2.0:
```
Comment (klust) [inline docs/2022-isc22/troubleshooting.md:364]: This should be 11.2.0 here and in the next lines for the setup that we have for ISC'22.

Comment (klust) [inline docs/2022-isc22/troubleshooting.md:364]: This should be 11.2.0 here and in the next lines for the setup that we have for ISC'22.

File: docs/2022-isc22/troubleshooting.md [hunk 3]
```diff
@@ -1 +1,485 @@
...
+    After doing so, **you don't have to redo the installation
+    from scratch**, you can use the `--module-only` option to only run the
+    sanity check and generate the module file again:
+    ```
+    eb subread.eb --module-only
+    ```
+
+---
+
+In the end, you should be able to install Subread 2.0.1 with the GCC 10.2.0 toolchain by fixing the problems with the `subread.eb` easyconfig file.
```
Comment (klust) [inline docs/2022-isc22/troubleshooting.md:473]: with the GCC 11.2.0 toolchain instead of 10.2.0.

Comment (klust) [inline docs/2022-isc22/troubleshooting.md:473]: with the GCC 11.2.0 toolchain instead of 10.2.0.

**Review by klust — CHANGES_REQUESTED** [review]

File: docs/2022-isc22/installing_software.md [hunk 4]
```diff
@@ -1 +1,465 @@
...
+```
+
+Loading the module for BCFtools changes that:
+
+```shell
+$ module load BCFtools/1.14-GCC-11.2.0
+
+$ module list
+Currently Loaded Modules:
+  1) EasyBuild/4.4.5                7) XZ/5.2.5-GCCcore-11.2.0
```
[the remark this answers is not in the document]
Comment (SebastianAchilles) [reply docs/2022-isc22/installing_software.md]: Updated in https://github.com/easybuilders/easybuild-tutorial/pull/93/commits/d8546b1ba5bbbea229cb4b582998cc7bebddaa41

File: docs/2022-isc22/installing_software.md [hunk 5]
```diff
@@ -1 +1,465 @@
...
+
+## Exercises
+
+***Guidelines***
+
+Do yourself a favor: don't peek at the solution until you have made an attempt to solve the exercise yourself!
+
+Please do not spoil solutions for others before they have been discussed by the tutorial organisers.
+
+The exercises are based on the easyconfig files included with EasyBuild 4.4.5.
```
[the remark this answers is not in the document]
Comment (SebastianAchilles) [reply docs/2022-isc22/installing_software.md]: Updated in https://github.com/easybuilders/easybuild-tutorial/pull/93/commits/d8546b1ba5bbbea229cb4b582998cc7bebddaa41

**Comment by ocaisa** [comment]
@klust When you give this the ok I am happy to merge

**Comment by ocaisa** [comment]
I'm going to merge this...

**Review by ocaisa — APPROVED** [review]

**Review by ocaisa — APPROVED** [review]

**Pull request merged**